src/client/app/redux/selectors/adminSelectors.ts

```diff
@@ -35,11 +33,13 @@ export const selectPossibleGraphicUnits = createAppSelector(
 /**
  * Calculates the set of all possible meter units for a meter.
  * This is any unit that is of type meter.
+ * @param state # TODO
```

Comment:
Can this TODO be completed?

Reply:
I wrote up documentation on state and locale for that function, let me know if it's satisfactory or needs changes.